app/controllers/posts_controller.rb

```diff
@@ -15,6 +15,6 @@ def index
   private
 
   def post_params
-    params.require(:post).permit(:message)
+    params.require(:post).permit(:message, created_at: Time.now)
```

Comment:
I don't think you need to add this to the permit, as the timestamp will be added at the model layer.